lib/document.js

```diff
@@ -2406,6 +2406,10 @@ Document.prototype.$__validate = function(pathsToValidate, options, callback) {
       (typeof options === 'object') &&
       ('validateModifiedOnly' in options);
 
+  const pathsToSkip = options &&
+      (typeof options === 'object') &&
```

Comment:
Careful, remember you wrote about [typeof object](https://masteringjs.io/tutorials/fundamentals/typeof-object) and why using this to check if a value is an object is dangerous.

I'd instead do `const pathsToSkip = get(options, 'pathsToSkip', null)`